ios/engine/KMEI/KeymanEngine/Classes/Manager.swift

```diff
@@ -390,7 +390,7 @@ public class Manager: NSObject, HTTPDownloadDelegate, UIGestureRecognizerDelegat
                   languageID: languageId,
                   languageName: languageName,
                   version: version,
-                  isRTL: false,
+                  isRTL: false, // rrb: how do we know this? possible source of losing RTL flag?
```

Comment:
If `isRTL` isn't defined in kmp.json, it's assumed to be false.

Reply:
Now I check the kmp.json, use the value or default to false.